.circleci/config.yml

```diff
@@ -197,13 +197,19 @@ commands:
           destination: results
       - run:
           working_directory: integration-tests
-          name: Slack notification on dev branch only
+          name: Run wait-for-job-finish script on dev branch only
```

Comment:
break apart single step. It'd be easier to find out which one if fails.

Reply:
I don't think these need to be broken up. We can design for the common case (this works as expected) vs the rare case (we are actively debugging this).